app/components/select-percentage.tsx

```diff
@@ -0,0 +1,161 @@
+          <div className="w-full rounded-full flex flex-row items-center justify-between space-x-2 bg-[#100827]">
+            <input
+              type="text"
+              onChange={handleAmountChange}
+              className="placeholder:text-[#433B5A] text-[#F9F9F9] bg-transparent outline-none focus:outline-none w-full p-4"
+              placeholder="Input how much you want to auto-swap"
+            />
+            <div className="w-fit h-full flex-none pr-4">
+              <img src="/pencil-edit.svg" alt="pencil edit" className="w-full h-auto" />
+            </div>
+          </div>
+
+          <fieldset className="w-full flex flex-col space-y-1">
+            <legend className="font-[400] text-[14px] text-[#F9F9F9]">
+              Select the percentage you want to auto-swap
+            </legend>
+            <div className="w-full flex gap-3 pt-4">
+              {[
+                { label: "25%", value: 25, id: "twenty-five" },
+                { label: "50%", value: 50, id: "fifty" },
+                { label: "75%", value: 75, id: "seventy-five" },
+                { label: "100%", value: 100, id: "hundred" },
+              ].map(({ label, value, id }) => (
+                <div
+                  key={id}
+                  className={`w-full flex items-center justify-center mx-auto`}
+                >
+                  <input
+                    type="radio"
+                    id={id}
+                    name="radio-group"
+                    value={value}
+                    className={`hidden peer`}
+                    onClick={() => setPercent(value as PercentageValue)}
+                  />
+                  <label
+                    htmlFor={id}
+                    className={`bg-[#100827] text-[#F9F9F9] text-[14px] font-[600] text-center w-full hover:cursor-pointer py-4 rounded-full border ${percent == value ? "border-blue-500" : "border-transparent"} peer-checked:border-blue-500`}
+                  >
+                    {label}
+                  </label>
+                </div>
+              ))}
+            </div>
+          </fieldset>
+
+          <div className="w-full flex flex-row items-center justify-start space-x-2">
+            <img
+              src="/time-half-pass.svg"
+              alt="time half pass"
+              className="w-auto h-auto"
+            />
+            <span className="text-[13px] text-[#F9F9F9] text-[400]">
+              Base Token
+            </span>
+          </div>
+
+          <div className="w-full flex gap-8 sm:gap-16">
+            {[
+              { logo: "/usdt-logo.png", label: "Usdt", value: "usdt", id: "usdt" },
+              { logo: "/usdc-logo.png", label: "Usdc", value: "usdc", id: "usdc" },
+            ].map(({ logo, label, value, id }) => (
+              <div
+                key={id}
+                className={`w-full flex items-center justify-center mx-auto`}
+              >
+                <input
```

Comment:
The cursor of the base tokens should be a pointer on hover. Also a base token should unselect when it has been previously selected and clicked again.